js/src/components/modals/SemRushUpsellAlert.js

```diff
@@ -0,0 +1,42 @@
+/* External dependencies */
+import React from "react";
+import { __, sprintf } from "@wordpress/i18n";
+
+/* Internal dependencies */
```

Comment:
This comment should be `/* Yoast dependencies */` while the "Internal dependencies" should be used for local components (the ones into this plugin).
I noticed similar inconsistency also in other SEMrush-related files, will fix.